index.html

```diff
@@ -50,5 +50,173 @@ <h1>I’m Mohammed Glad to see you!</h1>
         </ul>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="card1 card-work">
+        <img src="/img/Snapshoot-Portfolio.svg" alt="" />
+        <h1>Tonic</h1>
+        <div class="frame2">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="programing-lamguages">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </div>
+
+      <div class="card2 card-work">
+        <img src="/img/work2.svg" alt="" />
+        <h1>Multi-Post Stories</h1>
+        <div class="frame2">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="programing-lamguages">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </div>
+
+      <div class="card3 card-work">
+        <img src="/img/work3.svg" alt="" />
```

Comment:
- [ ] Good work so far. Make sure that you add all your image illustrations as a `background image` and not in the `img` tag. Please, let's rock it